### DISCUSSION TITLE
As secretary, I want to be able to assign testimonies to subject matter experts

### DISCUSSION THREAD

**Discussion by michnotmeesh** [a-doe-bo/hi-doe-bill-tracker#27] [General]
As secretary, I want to be able to assign testimonies to subject matter experts.

1. Log in
2. Navigate to "Saved Bills" and choose a bill
3. Will have an option to "Assign to Expert"
4. Dropdown of folders with experts sorted by department
5. Email templates (i.e., different hearings info) will pop up
6. Edit template (if necessary)
7. Send the email
8. "Assign to Expert" will grey out and secretary will know that the bill was assigned

**Comment by adrianwtam** [comment]
Can't experts exist outside of HIDOE?
It's impossible for just Government Workers to have a good grasp on every single subject.
These foreign experts won't have HIDOE accounts and can't be registered
Thus a Manual Form to type in their info will have to be created alongside the fast search on your proposal
